lib/modules-preprocessor.js

```diff
@@ -15,20 +15,32 @@ module.exports = class ModulesPreprocessor {
   toTree(inputTree, path) {
     if (path !== '/') { return inputTree; }
 
-    let merged = new MergeTrees([inputTree, this.getModulesTree()], { overwrite: true });
-
-    // Exclude the individual CSS files – those will be concatenated into the styles tree later
-    return new Funnel(merged, { exclude: ['**/*.' + this.owner.getFileExtension()] });
+    return new MergeTrees(
+      [
+        new Funnel(inputTree, {
+          exclude: ['**/*.' + this.owner.getFileExtension()]
+        }),
+        this.getModulesTree()
+      ],
+      { overwrite: true }
+    );
```

Comment:
Style files apparently now can live _anywhere_ in the `src` directory and will get concatenated into `vendor.css`. This is not explicitly speced anywhere yet, and the MU specs are rather vague about CSS in general.

Reply:
Me neither. 😂 

I will have another look at this soon.

Reply:
Just looked at this again. You are completely right. By applying this change, all processed files were just passed through and then concatenated by ember-cli. I mistook this for the output of `OutputStylesPreprocessor`.

Undoing this change highlights the actual problem: For some reason `OutputStylesPreprocessor#toTree` is only called for the dummy application, but never the actual addon itself.

This pretty much sound like a case for `addon/styles/.placeholder` 😂

Reply:
FFS. Creating `addon/styles/.placeholder` fixes it. And here I was, trying to create files in `src/*` like a mad men. 😂 

I guess this is a bug upstream, but at least this unblocks us for now.